pix2pix-ultimate/shared.py

```diff
@@ -1,12 +1,13 @@
 from __future__ import division
+import numpy as np
 import os
 import scipy.misc
 import time
-import numpy as np
 import tensorflow as tf
 from glob import glob
-from six.moves import xrange
+from metrics import *
```

Comment:
Explicitly stating what is going to be imported from a module makes it easier to know where things come from when reading the code :)